src/core/files.js

```diff
@@ -58,26 +61,10 @@ export class FileStore {
    *
    * @param   {Query}                 [query]                                   Query used to filter result.
    * @param   {Object}                [options]                                 Options
-   * @param   {Properties}            [options.properties]                      Custom properties to send with
-   *                                                                            the request.
-   * @param   {Number}                [options.timeout]                         Timeout for the request.
-   * @param   {Boolean}               [options.tls]                             Use Transport Layer Security
-   * @param   {Boolean}               [options.download]                        Download the files
-   * @return  {Promise}                                                         Promise
-   *
-   * @example
-   * var filesStore = new Kinvey.FilesStore();
-   * var query = new Kinvey.Query();
-   * query.equalTo('location', 'Boston');
-   * files.find(query, {
-   *   tls: true, // Use transport layer security
-   *   ttl: 60 * 60 * 24, // 1 day in seconds
-   *   download: true // download the files
-   * }).then(function(files) {
-   *   ...
-   * }).catch(function(err) {
-   *   ...
-   * });
+   * @param   {boolean}               [options.tls=true]                        Use transport layer security
```

Comment:
A good API reference will describe what the developer should expect when they choose different values for a property, particularly a boolean flag. So for example, you could say - 
```
By default, all requests to retrieve files are made with https.
By setting this flag to false, files are retrieved over unsecure http connections. This is not recommended because ...
```
I've provided a generic example which may not be accurate here, but the idea is to describe what the parameter *does* rather than just describe what the name *stands for*

Reply:
Good point. I will make the change.